packages/neon-core/src/rpc/Query.ts

```diff
@@ -167,9 +167,9 @@ export class Query {
   /**
    * This Query returns information about a specific transaction in either hexstring or human readable JSON.
    * @param txid hash of the specific transaction.
-   * @param verbose 0 for hexstring, 1 for JSON. Defaults to 1.
+   * @param verbose 0 for hexstring, 1 for JSON. Defaults to 0.
    */
-  public static getRawTransaction(txid: string, verbose: number = 1): Query {
+  public static getRawTransaction(txid: string, verbose: number = 0): Query {

```

Comment:
If you are going to narrow the type for getBlock, then you should be consistent and narrow it for all verbose param.

Reply:
I think it's better to keep `Query` class to the raw level and only narrow types for RPCClient.

Reply:
im talking about `verbose: 0|1`

Reply:
Fixed